src/styles/style.css

```diff
@@ -0,0 +1,94 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,300;1,400&family=Nunito:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,200;1,300;1,400;1,600;1,900&family=Plus+Jakarta+Sans:ital,wght@0,200;0,300;0,400;0,600;0,700;1,200;1,500&family=Poppins:ital,wght@0,200;0,400;0,600;1,100;1,200;1,300;1,400&family=Roboto:ital,wght@0,300;0,400;1,100;1,300&display=swap');
+
+* {
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+body {
+  background-color: rgba(236, 231, 231, 0.836);
+}
+
+.heading {
+  border-bottom: 3px solid rgb(223, 219, 219);
+  padding: 15px;
+}
+
+input {
+  border: none;
+  outline: none;
+  font-size: 20px;
+}
+
+.save-todo {
+  width: 93%;
+  padding-left: 5px;
+}
+
+.checkbox-task {
+  width: 93%;
+  padding-left: 5px;
+}
+
+.checkbox-task.checked {
+  text-decoration: line-through;
+  color: blue;
+}
+
+.list-cont {
+  display: flex;
+}
+
+#enter-todo {
+  width: 100%;
+  font-style: italic;
+}
+
+.todolist {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 10%;
+}
+
+.list-container {
+  background-color: white;
+  box-shadow: 10px 5px 10px 5px grey;
+  width: 50%;
+}
+
+.list-group {
+  border-bottom: 3px solid rgb(223, 219, 219);
+  padding: 20px;
+  font-size: 20px;
+  font-weight: 400;
+}
+
+.clear {
+  background-color: rgb(230, 224, 224);
+  padding: 5px;
+  text-align: center;
+  cursor: pointer;
+}
+
+.bin {
+  float: right;
+  display: none;
+}
+
+.menu {
+  float: right;
+}
+
+.refresh {
+  float: right;
+}
```

Comment:
Your CSS looks perfect , but it could be better if you avoid duplication. It will make your code look Dry .smirk_cat You can correct that by using a common class to the two elements.